src/EditorFeatures/Core/Interactive/InteractiveSession.cs

```diff
@@ -194,6 +199,18 @@ private void AddSubmissionProjectNoLock(ITextBuffer submissionBuffer, string lan
             _currentSubmissionProjectId = ProjectId.CreateNewId(newSubmissionProjectName);
             var newSubmissionDocumentId = DocumentId.CreateNewId(_currentSubmissionProjectId, newSubmissionProjectName);
 
+            // Associate a name with both the editor document and our roslyn document so LSP can make requests on it.
+            var extension = languageName switch
+            {
+                LanguageNames.CSharp => "cs",
+                LanguageNames.VisualBasic => "vb",
+                _ => throw new ArgumentException($"Unexpected language name {languageName}")
+            };
+            var submissionFilePath = Path.Combine(Path.GetTempPath(), $"Submission{SubmissionCount}.{extension}");
```

Comment:
>submissionFilePath [](http://example.com/codeflow?start=16&length=18)

nit: call the variable `newSubmissionFilePath` for consistency with similar variables.